python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/logging.schema.yml

```diff
@@ -33,6 +33,10 @@ keys:
             valid_values: ["alerts", "critical", "debugging", "emergencies", "errors", "informational", "notifications", "warnings", "disabled"]
             description: |
               Buffer logging severity level.
+      repeat_messages:
+        type: bool
+        description: |
+          Summarize concurrent repeat messages
```

Comment:
```suggestion
        description: Summarize concurrent repeat messages.
```

Reply:
We use multi-line pipe in multiline strings for readability and proper formatting. So here its not required.
Regarding the existing ones, if its a lot of changes you can ignore it in this PR and just fix the new changes.

Reply:
Done, as well as the existing ones in the same file.